client/pages/sections/protocols/sections.js

```diff
@@ -106,13 +110,68 @@ const getBadges = (section, newComments, values) => {
 
   const fields = getFieldKeys(section, values);
 
+  // Initialize groups for fields with and without values
+  const fieldsWithValues = [];
+  const fieldsWithoutValues = [];
+
+  section.fields?.forEach((field, index) => {
+    console.log("Processing field:", field.name);
```

Comment:
These logs are going to be very noisy in opensearch when doing server-side rendering of the page, and could potentially be logging PII which would be a security risk
```suggestion
```